src/client/app/+graph/graph.component.ts

```diff
@@ -11,4 +13,19 @@ import { CanvasComponent } from '../shared/index';
   styleUrls: ['graph.component.css'],
   directives: [CanvasComponent]
 })
-export class GraphComponent { }
+export class GraphComponent implements OnInit {
+
+  graphId: string;
+
+  constructor(private googleRealtimeService: GoogleRealtimeService,
+              private route: ActivatedRoute) {
+  }
+
+  ngOnInit(): void {
+    this.route.params.forEach((params: Params) => {
```

Comment:
Well [ActivatedRoute](https://angular.io/docs/ts/latest/api/router/index/ActivatedRoute-interface.html) is an Observable by API, and forEach() is similar to subscribe().

I think both work. Why did I use forEach() instead of subscribe() here? No idea, I just kind of made it up. We can switch to subscribe() later if we feel like unsubscribing maybe?